src/instruction/utils.ts

```diff
@@ -1,39 +1,83 @@
-import { BN } from "@project-serum/anchor";
+import { web3, BN } from "@project-serum/anchor";
 
-export function convertNumbersToBNs(data: Object | any[], keysToUpdate?: string[]) {
+export function convertNumbersToBNs(
+  data: Object | any[],
+  keysToUpdate?: string[]
```

Comment:
Let's make this `keysToUpdate: string[] = []` instead. This makes the body of the function less complex by having all the requirements of the parameters defined in the signature instead of also in the body, all without changing how the function can be called.

Reply:
Updated 👍